mocks/DDtheta_mocks/countpairs_theta_mocks_impl.c.src

```diff
@@ -94,63 +94,64 @@ countpairs_theta_mocks_func_ptr_DOUBLE countpairs_theta_mocks_driver_DOUBLE(cons
 
     /* Array of function pointers */
     countpairs_theta_mocks_func_ptr_DOUBLE allfunctions[] = {
-#ifdef __AVX512F__
-        countpairs_theta_mocks_avx512_instrinsics_DOUBLE,
+        countpairs_theta_mocks_fallback_DOUBLE,
+#ifdef __SSE4_2__
+        countpairs_theta_mocks_sse_intrinsics_DOUBLE,
 #endif
 #ifdef __AVX__
-        countpairs_theta_mocks_avx_instrinsics_DOUBLE,
+        countpairs_theta_mocks_avx_intrinsics_DOUBLE,
```

Comment:
@lgarrison Good catch! I am glad that the typo was consistent. Glad that I always copy-paste from the actual kernel function...pheww..